.github/workflows/library-ci.yml

```diff
@@ -0,0 +1,167 @@
+name: CI
+
+on:
+  workflow_call:
+    inputs:
+      mvn-verify-opts:
```

Comment:
So you added this option but no where use that.

Reply:
Yes, not sure which options to use, to be honest, but I've added just to make sure that we can use this in case we need.
Like we use for some of the services (Example template):
```
    with:
      mvn-verify-opts: |
        [
          "-pl :service-name -Denforcer.skip=true -Pcoverage",
          "-pl :service-name1,:service-name2 -Denforcer.skip=true -Pcoverage",
          "-pl :service-name -Denforcer.skip=true -Pcoverage"
        ]
```